portfolio/src/main/webapp/index.html

```diff
@@ -124,35 +124,38 @@ <h2>Education</h2><br>
         <hr>
 
         <h3>Leave a comment!</h3>
-        <script>
-            checkLoginStatus();
-        </script>
+        <div id="display-container">
```

Comment:
I think its best to exclude javascript functions in the html. I think you can trigger the function by an onload call.

https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Reply:
Very good point! The function is added as an onload call.